### PR TITLE
Validate that ext-zip is available in pm:security-php.

### PR DESCRIPTION
**Pull request by weitzman**

### CODE REVIEW

File: src/Commands/pm/SecurityUpdateCommands.php [hunk 1]
```diff
@@ -154,14 +154,12 @@ protected function calculateSecurityUpdates($composer_lock_data, $security_advis
...
      *
-     * Thanks to https://github.com/FriendsOfPHP/security-advisories
-     * and Symfony for providing this service.
-     *
      * @param array $options
      *
      * @return UnstructuredData
      * @throws \Exception
      * @command pm:security-php
+     * @validate-php-extension ext-zip
```
Comment (joestewart) [inline src/Commands/pm/SecurityUpdateCommands.php]: Composer won't allow installation at all if the zip extension isn't installed will it?

https://github.com/drush-ops/drush/blob/10.x/composer.lock#L1077

Comment (weitzman) [reply src/Commands/pm/SecurityUpdateCommands.php]: True, but one can use --ignore-platform-reqs to bypass.

Comment (weitzman) [reply src/Commands/pm/SecurityUpdateCommands.php]: Its too bad Enlightn bakes in this Zipfile cache. Would be better to factor that out as an optional dependency.

**Comment by paras-malhotra** [comment]
@weitzman, the Enlightn security checker (from v1.8.0) no longer requires `ext-zip` via https://github.com/enlightn/security-checker/pull/19.

**Comment by weitzman** [comment]
Thanks much. Drush PR at https://github.com/drush-ops/drush/pull/4732. Will merge soon.

**Pull request merged**